docs/api/net.md

```diff
@@ -62,3 +62,13 @@ Creates a [`ClientRequest`](./client-request.md) instance using the provided
 `options` which are directly forwarded to the `ClientRequest` constructor.
 The `net.request` method would be used to issue both secure and insecure HTTP
 requests according to the specified protocol scheme in the `options` object.
+
+### `net.isOffline()`
```

Comment:
Prefer positive conditions, i.e. this should be `net.isOnline()` (this would also match the `navigator.onLine` API that @jkleinsc mentioned)

Reply:
It should, IMHO, also be a property, not a method. (https://github.com/electron/electron/issues/12745)

Reply:
changed to `net.isOnline()`